tfhe/docs/getting_started/quick_start.md

```diff
@@ -64,98 +44,101 @@ fn main() {
 }
 ```
 
-#### Boolean example
+Default configuration for x86 Unix machines:
+```toml
+tfhe = { version = "0.3.0", features = ["integer", "x86_64-unix"]}
+```
+
+Other configurations can be found [here](../getting_started/installation.md).
+
+### Imports.
+
+`tfhe` uses `traits` to have a consistent API for creating FHE types and enable users to write generic functions. To be able to use associated functions and methods of a trait, the trait has to be in scope.
+
+To make it easier, the `prelude` 'pattern' is used. All `tfhe` important traits are in a `prelude` module that you **glob import**. With this, there is no need to remember or know the traits to import.
+
+```rust
+use tfhe::prelude::*;
+```
+
+### 1. Configuring and creating keys.
+
+The first step is the creation of the configuration. The configuration is used to declare which type you will use or not use, as well as enabling you to use custom crypto-parameters for these types for more advanced usage / testing.
+
+Creating a configuration is done using the ConfigBuilder type.
 
-Here is an example of how the library can be used to evaluate a Boolean circuit:
+In this example, 8-bit unsigned integers with default parameters are used. The `integers` 
+feature must also be enabled, as per the table on the [Getting Started page](../getting_started/installation.md).
 
-{% hint style="warning" %}
-Use the `--release` flag to run this example (eg: `cargo run --release`)
-{% endhint %}
+The config is done by first creating a builder with all types deactivated. Then, the `uint8` type with default parameters is activated.
```

Comment:
```suggestion
The config is done by first creating a builder with all types deactivated. Then, the integers type with default parameters are activated, since we are going to use FheUint8 values.
```